.github/workflows/test-spras.yml

```diff
@@ -126,3 +126,12 @@ jobs:
         tags: latest
         cache_froms: reedcompbio/mincostflow:latest
         push: false
+    - name: Build AllPairs Docker image
+      uses: docker/build-push-action@v1
+      with:
+        path: docker-wrappers/AllPairs/.
+        dockerfile: docker-wrappers/AllPairs/Dockerfile
+        repository: annaritz/allpairs
```

Comment:
One of us will need to move it to the reedcompbio organization before finalizing the pull request